Added Hacktoberfest Contributing Guidelines (#6509)
appwrite/appwrite: HACKTOBERFEST.md

# Appwrite hacktoberfest contribution

Welcome to the Appwrite Hacktoberfest contribution! We appreciate your interest in contributing to our open-source project. Please read this carefully to understand how to get started and make your contributions count.

## Familiarize with the project

> - Before you start contributing, it's essential to get familiarized with our project. You can do that by following the [main README](https://github.com/appwrite/appwrite/blob/main/README.md).

Reviewer: It doesn't seem like this needs to be a bullet. Also, some Grammarly suggestions

```suggestion
Before you start contributing, familiarize yourself with our project by reading the [main README](https://github.com/appwrite/appwrite/blob/main/README.md).
```

> ## Code of Conduct

Reviewer: Sentence case

> - Please ensure that you follow our [Code of Conduct](https://github.com/appwrite/awesome-appwrite/blob/master/CODE_OF_CONDUCT.md) at all times. We aim to maintain a respectful and inclusive community.

Reviewer: It doesn't seem like this needs to be a bullet. Also, some Grammarly suggestions.

```suggestion
Please ensure that you always follow our [Code of Conduct](https://github.com/appwrite/awesome-appwrite/blob/master/CODE_OF_CONDUCT.md). We aim to maintain a respectful and inclusive community.
```

> ### Contributing Guide

Reviewer: Sentence case. Also, I'm not sure why this is a level 3 heading instead of 2 🧐

> Please review our [Contributing Guide](https://github.com/appwrite/appwrite/blob/main/CONTRIBUTING.md) before contributing. We strive to maintain a friendly and inclusive community for everyone.

Reviewer: > We strive to maintain a friendly and inclusive community for everyone.

The contributing guide is more about contributing by giving architecture information and instructions on how to update code, run tests, submit a PR, etc, rather than being inclusive.

> ## Hacktoberfest Contribution Guidelines

Reviewer: Sentence case

> ## Hacktoberfest Contribution Guidelines

Reviewer: Sentence case

To participate in Hacktoberfest with Appwrite, follow these guidelines:

> 1. **Valid Issues**: To make your contribution count, please look for issues labeled with `hacktoberfest`. Only issues labeled with `hacktoberfest` will count as a valid contribution.

Reviewer: I'm not sure if the bold part needs to be sentence case too.

2. **Wait for Issue Assignment**: After you have identified an issue to work on, please wait for it to be assigned to you by our team. We assign issues on a first-come, first-serve basis.

> 3. **Label Your Pull Requests**: Appwrite does not have the `hacktoberfest` label in our repositories, but we use it on specific issues. For your pull request to be valid, our team will review and add the `hacktoberfest-accepted` label once it meets the criteria.

Reviewer: > Label Your Pull Requests

They aren't labeling anything so this heading may not make sense.

> 4. **Discuss New Issues**: If you're interested in finding and adding new issues for Hacktoberfest, please discuss it with the team on our Discord server: [appwrite.io/discord](https://appwrite.io/discord) in the `#hacktoberfest` channel.

Reviewer: ```suggestion
4. **Discuss New Issues**: If you're interested in finding and adding new issues for Hacktoberfest, please discuss it with the team on our [Discord server](https://appwrite.io/discord) in the `#hacktoberfest` channel.
```

5. **Be Patient with PR Reviews**: PR reviews may take up to 15 days due to the volume of contributions. However, we also host PR review parties every week where you can add your PRs for instant review.

>  

Reviewer: extra line

```suggestion
```

> 6. **Regular Updates**: If you've been assigned an issue, it's essential to share updates every 3 days. Failure to do so may result in the unassignment of the issue.

Reviewer: ```suggestion
6. **Regular Updates**: If you've been assigned an issue, it's essential to share updates every 3 days. Failure to do so may result in being unassigned from the issue.
```

> ## Getting Started

Reviewer: Sentence case

1. Fork the Appwrite repository you'd like to contribute to.

2. Clone your forked repository to your local machine.

   ```bash

>       git clone https://github.com/your-username/repository-name.git

Reviewer: There's an extra indentation here. The code should line up with the 1st backtick
